core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -13,74 +22,93 @@
 import com.novoda.noplayer.VideoContainer;
 import com.novoda.noplayer.VideoDuration;
 import com.novoda.noplayer.VideoPosition;
+import com.novoda.noplayer.exoplayer.forwarder.ExoPlayerForwarder;
 import com.novoda.noplayer.player.PlayerInformation;
 
 import java.util.Collections;
 import java.util.List;
 
 public class ExoPlayerTwoImpl extends PlayerListenersHolder implements Player {
```

Comment:
Facade became a wrapper during out refactorings so quite a few of the methods here are just inlines.